administration/monitoring.md

```diff
@@ -80,132 +93,150 @@ $ curl -s http://127.0.0.1:2020 | jq
-| fluentbit\_input\_storage\_chunks\_busy         | name: the name or alias for the input instance  | "Busy" chunks are chunks that are being processed/sent by outputs and are not eligible to have new data appended.                                                        | gauge   | chunks  |
-| fluentbit\_input\_storage\_chunks\_busy\_bytes  | name: the name or alias for the input instance  | The sum of the byte size of each chunk which is currently marked as busy.                                                                                                | gauge   | bytes   |
-|                                                 |                                                 |                                                                                                                                                                          |         |         |
-| fluentbit\_output\_upstream\_total\_connections | name: the name or alias for the output instance | The sum of the connection count of each output plugins.                                                                                                                  | gauge   | bytes   |
-| fluentbit\_output\_upstream\_busy\_connections  | name: the name or alias for the output instance | The sum of the connection count in a busy state of each output plugins.                                                                                                  | gauge   | bytes   |
-
-### Uptime Example
+### REST API interface
+
+Fluent Bit exposes the following endpoints for monitoring.
+
+| URI                        | Description   | Data format           |
+| -------------------------- | ------------- | --------------------- |
+| /                          | Fluent Bit build information. | JSON |
+| /api/v1/uptime             | Return uptime information in seconds. | JSON |
+| /api/v1/metrics            | Display internal metrics per loaded plugin. | JSON                  |
+| /api/v1/metrics/prometheus | Display internal metrics per loaded plugin in Prometheus Server format. | Prometheus Text 0.0.4 |
+| /api/v1/storage            | Get internal metrics of the storage layer / buffered data. This option is enabled only if in the `SERVICE` section of the property `storage.metrics` is enabled. | JSON |
+| /api/v1/health             | Display the Fluent Bit health check result. | String |
+| /api/v2/metrics            | Display internal metrics per loaded plugin. | [cmetrics text format](https://github.com/fluent/cmetrics) |
+| /api/v2/metrics/prometheus | Display internal metrics per loaded plugin ready in Prometheus Server format. | Prometheus Text 0.0.4 |
+| /api/v2/reload             | Execute hot reloading or get the status of hot reloading. See the [hot-reloading documentation](hot-reload.md). | JSON |
+
+### v1 metrics
+
```

Comment:
should be an sentence here between headers but it's not a blocker, same for below, but that can totally be iterated later.

Reply:
ACK -- will add a sentence there 👍